src/rating-menu/rating-menu.ts

```diff
@@ -34,16 +34,12 @@ export type RatingMenuState = {
         <symbol
           id="ais-StarRating-starSymbol"
           viewBox="0 0 24 24"
-          width="24"
-          height="24"
```

Comment:
width=height=1em will help it not to be massive, even if the css isn't used

Reply:
off record, we discussed that width and height were missing on the non hidden `<svg />` tags, so I added that here 0a79c49